site/_posts/2016-10-13-announcing-euwe-1-beta1.3.html.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Announcing Euwe-1-Beta1.3 
+author: jprause
+date: 2016-10-13 12:49:24 UTC
+comments: true
+published: true
+tags: releases announcements
+---
+
+On behalf of the ManageIQ team, we're pleased to announce the Beta release for ManageIQ Euwe! 
+
+Euwe is the next milestone release for ManageIQ cloud and virtualization management platform. With each release Manageiq gets more robust and feature complete across providers. In this release, we not only attended to providers and chargeback, but we also made some essential improvements to the appliance itself including performance, and high availability.
```

Comment:
Insert comma after "With each release"

remove comma after "performance" in the last sentence.